rest/index.py

```diff
@@ -468,6 +481,433 @@ def createIncident():
     return response
 
 
+@post("/alertstatus")
+@post("/alertstatus/")
+def update_alert_status():
+    """Update the status of an alert.
+
+    Requests are expected to take the following (JSON) form:
+
+    ```
+    {
+        "alert": str,
+        "status": str,
+        "user": {
+            "email": str,
+            "slack": str
+        },
+        "identityConfidence": str
+        "response": str
+    }
+    ```
+
+    Where:
+        * `"alert"` is the unique identifier fo the alert whose status
+        we are to update.
+        * `"status"` is one of "manual", "inProgress", "acknowledged"
+        or "escalated".
+        * `identityConfidence` is one of "highest", "high", "moderate", "low",
+        or "lowest".
+
+
+    This function writes back a response containing the following JSON.
+
+    ```
+    {
+        "error": Optional[str]
+    }
+    ```
+
+    If an error occurs and the alert is not able to be updated, then
+    the "error" field will contain a string message describing the issue.
+    Otherwise, this field will simply be `null`.  This function will,
+    along with updating the alert's status, append information about the
+    user and their response to `alert['details']['triage']`.
+
+    Responses will also use status codes to indicate success / failure / error.
+    """
+
+    response.content_type = "application/json"
+
+    initConfig()
+
+    mongo = MongoClient(options.mongohost, options.mongoport)
+    alerts = mongo.meteor["alerts"]
+
+    try:
+        req = json.loads(request.body.read())
+        request.body.close()
+    except ValueError:
+        response.status = StatusCode.BAD_REQUEST
+        response.body = json.dumps({"error": "Missing or invalid request body"})
```

Comment:
In all of the cases in this file where `response.body` is set to a value that's returned from `json.dumps` I believe you can just use the [native bottle support for JSON by returning a dict](https://bottlepy.org/docs/dev/tutorial.html#generating-content).

```suggestion
        response.body = {"error": "Missing or invalid request body"}
```

I'll won't add this note to all the instances like this in the file but it should apply to all of them.